userenvs/stream9.json

```diff
@@ -0,0 +1,94 @@
+{
+  "workshop": {
+    "schema": {
+      "version": "2020.03.02"
+    }
+  },
+  "userenv": {
+    "name": "stream8",
```

Comment:
Should it be stream9?

Reply:
It should, editing fail on my part.  Nice catch!